src/components/ThemeRegistry/theme.js

```diff
@@ -61,12 +61,12 @@ const theme = createTheme({
       fontWeight: 700,
       fontFamily: montserrat.style.fontFamily,
     },
-    subheading: ({ theme }) => ({
+    subheading: {
       fontFamily: montserrat.style.fontFamily,
       fontSize: "1rem", // 16px
       fontWeight: 700,
-      color: theme.palette.orange.main,
-    }),
+      color: colors.orange.main,
```

Comment:
Use palette better to keep it decoupled from colors as possible.

Reply:
Looks like only this solution work